### PR TITLE
Consolidate mspIds ...

### PR DESCRIPTION
**Pull request by g2flyer**
.. so create-auction templates work not only with mock but also with fabric network

### CODE REVIEW

File: demo/client/backend/fabric-gateway/auction.json [hunk 1]
```diff
@@ -171,21 +171,21 @@
 		"id": 1,
 		"displayName": "A-Telecom",
 		"principal": {
-			"mspId": "org1",
+			"mspid": "Org1MSP",
```
[the remark this answers is not in the document]
Comment (g2flyer) [reply demo/client/backend/fabric-gateway/auction.json]: The reason for this all-lower-case version is "historical" I first fixed demo/scenario/Auctioneer1.createAuction.json which was all lower-case and then tried remove the reduncancy with demo/client/backend/fabric-gateway/auction.json  and demo/client/backend/mock/api/serverapi.json . Alas, that turned out to be a bit more complicated (e.g., due to docker i could not just trivially refer to  demo/scenario/Auctioneer1.createAuction.json by relative path or symlink) and so i gave up and had to restore auction.json, which i just copied from demo/scenario, not noticing the camel-case.  For consistency reason i'll change it, though, back here to camel case (and make Auctioneer1.createAuction.json also camel case ..)

[the remark this answers is not in the document]
Comment (g2flyer) [reply demo/client/backend/fabric-gateway/auction.json]: (fixed now)

[the remark this answers is not in the document]
Comment (g2flyer) [reply demo/client/backend/fabric-gateway/auction.json]: btw: i notice now that github shows this nicely in the diff whereas with 'git diff' by default it is much harder to spot. I guess i should make '--color-words' the default ....

**Comment by g2flyer** [comment]
> Would you apply this patch please...
> ...

Just did.

**Comment by mbrandenburger** [comment]
This PR may have a conflict with #205 as I changes some `mspid` to `mspId`. So I will merge this one here and rebase #205 again! Thanks!

**Pull request merged**